docs/src/tutorial/temporal_resolution.md

```diff
@@ -0,0 +1,175 @@
+## Introduction
+
+### Installation and upgrades
+
+If you haven't yet installed the tools yet, please follow the installation guides: 
+- For Spine Toolbox: [Spine Toolbox installation guide](https://github.com/spine-tools/SpineOpt.jl#installation)
+- For SpineOpt: [SpineOpt installation guide](https://github.com/spine-tools/Spine-Toolbox#installation)
+
+If you are not sure whether you have the latest version, please upgrade to ensure compatibility with this guide.
+
+- For Spine Toolbox:
+    - If installed with pipx, then use `python -m pipx upgrade spinetoolbox`
+    - If installed from sources using git, then `git pull`, `python -m pip install -U -r requirements.txt`
+
+- For SpineOpt: [SpineOpt upgrade guide](https://github.com/spine-tools/SpineOpt.jl#upgrading)
+
+### Model assumptions 
+
+- This tutorial builds upon the Simple System tutorial. Please follow [that tutorial](https://spine-tools.github.io/SpineOpt.jl/latest/tutorial/simple_system/) first, or download the [finished system](**NEEDS LINK**).
+- The scenario runs for 5 hours and the default temporal resolution is 1 hour.
+- The fuel_node has a temporal resolution of [1, 1, 1, 2] hours.
+
+## Tutorial
+
+### Adjustments to Simple System
+
+Before demonstrating flexible temporal resolution, we need to adjust the Simple System tutorial so the results are more interesting to examine.
+
+!!! note If you are ever missing a window in the Spine DB Editor, go to the *Hamburger menu* (top right) > under View: Docks > Choose the missing window - then drag and resize the window to your preference
+
+#### Adding variable electricity demand
+
+In SpineToolbox, double click on the *InputData*  block.
+
+In the *Object tree* window:
+- Expand "node"
+- Select the electricity node
+- In the *Object parameter value* window, find the value for the electricity node's demand (should be 150 from the Simple System) - right click > Edit\
+	`Parameter type: Time Series fixed resolution`\
+	Copy and paste (or fill in the table):
+	
+		10
+		20
+		40
+		50
+		20
+
+- Click OK - this sets the electricity demand to the variable profile listed above
+
+![image](figs_temporal_resolution/temporal_resolution_time_series_demand.PNG)
+
+### Uniform temporal resolution
+
+#### Setting model start & end timestamps
+
+In the *Object Tree* window:
+- Expand the model
+- Click on "simple"
+
+In the *Object parameter value* window:
+- Add model_start \
+	`parameter_name:` (Double click) `model_start`\
+	`alternative_name:` `Base`\
+	`value:` (Right click) Edit > `Parameter Type: Date Time` > `1 Jan` (default)
+
+- Add model_end\
+	`parameter_name: model_end`\
+	`alternative_name: Base`\
+	`value:` Edit > `Parameter Type: Date Time` > 5 hours after start
+
+Your *Object parameter value* window show look like this:
+
+![image](figs_temporal_resolution/temporal_resolution_model_start_end.PNG)
+
+!!! Regularly save your changes by selecting the *Hamburger menu* (top right) > Commit > Write a message, such as "Save" > OK
```

Comment:
Do you intend to write a note? In that case the keyword 'note' may be missing...

Reply:
Good catch.